cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/dao/messages/CassandraGroupedMessageFilter.java

```diff
@@ -33,7 +33,9 @@
 
 public class CassandraGroupedMessageFilter implements CassandraFilter<GroupedMessageBatchEntity> {
     private final String book, page, groupName;
-    private final Integer limit;
+
+    /** limit must be strictly positive ( limit greater than 0 ) */
```

Comment:
Let's check this condition in constructor

Reply:
I had to revert this change. checking in constructor doesn't fit well, because there are cases when limit is 0 and it means that it simply is not applied.